lib/tui/dune

```diff
@@ -11,4 +11,5 @@
   pretty
   shell
   tea
-  widget))
+  widget
+  init))
```

Comment:
I'm trying to sort dependencies alphabetically to search for it easier, so could we put `init` in the alphabetical order